tests/foreman/ui/test_contentcredential.py

```diff
@@ -5,9 +5,9 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: Hosts-Content
+:CaseComponent: ContentManagement
```

Comment:
How about CaseComponent `GPGKeys` ? as we have same name component in bugzilla ..

Reply:
@vijay8451 I can see why it would be in ``GPGKeys`` but that means someone who owns that component should take over and remove it from under my component ownership in ``rp_config.yaml`` file.  @pgagne you are the owner for ``GPGKeys``.  Would it make sense to put this under your ownership?

Reply:
@vijay8451 @latran Yes I think this would be better under GPGKey. Please update them

Reply:
@pgagne I'll take the liberty of updating this file to your component ownership as part of my changes :)